js/ui/camera.js

```diff
@@ -513,7 +515,7 @@ util.extend(Camera.prototype, /** @lends Map.prototype */{
      * @param {number} [options.screenSpeed] The average speed of the animation measured in screenfuls
      *     per second, assuming a linear timing curve. If `options.speed` is specified, this option is ignored.
      * @param {Function} [options.easing] An easing function for the animated transition.
-     * @param {EventData} [eventData] Data to propagate to any event listeners.
+     * @param {Object} [eventData] Data to propagate to any event listeners.
```

Comment:
`EventData` had been rendered meaningless, right? I had thought from prior conversation that this data you could send in is an arbitrary object. Is that wrong?